modules/ossm-federation-config-meshPeer.adoc

```diff
@@ -0,0 +1,111 @@
+* The names of the local ingress and egress gateways that is used for interactions with the specified peer mesh.
+* The client ID used by the other mesh when sending requests to this mesh.
+* The trust domain used by the other mesh.
+* The name of a `ConfigMap` containing a root certificate that is used to validate client certificates in the trust domain used by the other mesh.
+
+Meshes are federated on a one-to-one basis, so each pair of peers requires a `ServiceMeshPeer` file specifying the federation connection to the other service mesh. For example, federating three meshes named `red`, `blue`, and `green` would require six `ServiceMeshPeer` files.
+
+. On red-mesh-system, create a `ServiceMeshPeer` for the green mesh.
+. On red-mesh-system, create a `ServiceMeshPeer` for the blue mesh.
+. On green-mesh-system, create a `ServiceMeshPeer` for the red mesh.
+. On green-mesh-system, create a `ServiceMeshPeer` for the blue mesh.
+. On blue-mesh-system, create a `ServiceMeshPeer` for the red mesh.
+. On blue-mesh-system, create a `ServiceMeshPeer` for the green mesh.
+
+In the following example, the administrator for the `red-mesh` is configuring federation with the `green-mesh`.
+
+.Example ServiceMeshPeer resource for red-mesh
+[source,yaml]
+----
+kind: ServiceMeshPeer
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: green-mesh
+  namespace: red-mesh-system
+spec:
+  remote:
+    addresses:
+    - ingress-red-mesh.red-mesh-system.apps.domain.com
+  gateways:
+    ingress:
+      name: ingress-green-mesh
+    egress:
+      name: egress-green-mesh
+  security:
+    trust:
+      domain: green-mesh.local
+    clientID: green-mesh.local/ns/green-mesh-system/sa/egress-red-mesh-service-account
+    certificateChain:
+      kind: ConfigMap
+      name: green-mesh-ca-root-cert
+----
+
+.ServiceMeshPeer configuration parameters
+[options="header"]
+[cols="l, a, a"]
+|===
+|Parameter |Description |Values
+|metadata:
+  name:
+|Name of the peer mesh that this file is configuring federation with.
+|String
+
+|metadata:
+  namespace:
+|System namespace for this mesh, that is, where the mesh control plane is installed.
+|String
+
+|spec:
+  remote:
+    addresses:
+|List of addresses of the peer meshes' ingress gateways that are servicing requests from this mesh.
+|<IngressName>.<SystemName>.apps.domain.com
+
+|spec:
+  gateways:
+    ingress:
+      name:
+|Name of the ingress on this mesh that is servicing requests received from the peer mesh. For example, `ingress-green-mesh`.
+|
+
+|spec:
+  gateways:
+    egress:
+      name:
+|Name of the egress on this mesh that is servicing requests sent to the peer mesh. For example, `egress-green-mesh`.
+|
+
+|spec:
+  security:
+    trust:
+      domain:
+|The trust domain used by the peer mesh.
+|<peerMeshName>.local
+
+|spec:
+  security:
+    clientID:
+|The client ID used by the peer mesh when calling into this mesh.
+|<peerMesh>.local/ns/<peerMeshSystem>/sa/egress-this-mesh-service-account
```

Comment:
More accurate would be `<peerMeshTrustDomain>.local/ns/<peerMeshSystem>/sa/<peerMeshEgressGatewayName>-service-account